vite_ruby/lib/vite_ruby/manifest.rb

```diff
@@ -77,10 +77,10 @@ def lookup!(*args, **options)
   # Example:
   #   manifest.lookup('calendar.js')
   #   => { "file" => "/vite/assets/calendar-1016838bab065ae1e122.js", "imports" => [] }
-  def lookup(name, type: nil)
-    @build_mutex.synchronize { builder.build } if should_build?
+  def lookup(name, type: nil, force_build: false)
+    @build_mutex.synchronize { builder.build } if force_build || should_build?
```

Comment:
This is wrong, we don't want to force build in production for example.

The correct way should be something like `config.auto_build && (!dev_server_running? || force_build)`